lib/ssh/src/ssh_sftpd.erl

```diff
@@ -664,10 +664,11 @@ open(Vsn, ReqId, Data, State) when Vsn >= 4 ->
     do_open(ReqId, State, Path, Flags).
 
 do_open(ReqId, State0, Path, Flags) ->
-    #state{file_handler = FileMod, file_state = FS0, root = Root, xf = #ssh_xfer{vsn = Vsn}} = State0,
+    #state{file_handler = FileMod, file_state = FS0, xf = #ssh_xfer{vsn = Vsn}} = State0,
     XF = State0#state.xf,
```

Comment:
I was trying to not change things unrelated to the bug-fix. I will change it to State0#state.xf and will remote the FX variable.